Packages/com.unity.inputsystem/InputSystem/Editor/Internal/InputActionSerializationHelpers.cs

```diff
@@ -64,6 +64,30 @@ public static void AddActionMap(SerializedObject asset)
             mapProperty.FindPropertyRelative("m_Bindings").ClearArray();
         }
 
+        public static void AddActionMapFromObject(SerializedObject asset, InputActionMap map)
```

Comment:
`AddActionMapToActionAssetObject`

Reply:
There is "AddActionMap" that simply adds an empty map. "FromObject" means the given object will be added to the map. There is a method "AddActionMap" thus i called this one "AddActionMapFromObject"